src/main/java/Duke.java

```diff
@@ -1,10 +1,110 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
+        printBanner();
+        Scanner in = new Scanner(System.in);
+        String userInput;
+        ArrayList<Task> listOfTasks = new ArrayList<>();
+        boolean isRunning = true;
+
+        while (isRunning) {
+            userInput = in.nextLine();
+            String[] userInputSplit = userInput.split(" ");
+            isRunning = runCommand(userInput, listOfTasks, userInputSplit);
+        }
+    }
+
+
+    /**
+     * Run the command the user typed.
+     *
+     * @param userInput      What the user inputted into the command line.
+     * @param listOfTasks    An arraylist storing the list of tasks the user created.
+     * @param userInputSplit An array storing the user input but split by " ".
+     * @return false only if "bye" is typed. true otherwise.
+     */
+    private static boolean runCommand(String userInput, ArrayList<Task> listOfTasks, String[] userInputSplit) {
+        Task task;
+        String userInputNoCommand = userInput.replace(userInputSplit[0], "");
+        switch (userInputSplit[0]) {
+        case "bye":
+            System.out.println("Bye. Hope to see you again soon!");
```

Comment:
Would it be better to have a separate display class or something similar that will handle all CLI prints? (including logo print)